### PR TITLE
Updated the explanation of the watershed markers example.

### PR DESCRIPTION
**Pull request by oew1v07**
This fixes issue #1650 
Completed at the EuroScipy 2015 sprint by me and @dpshelio.
@emmanuelle: ping

### CODE REVIEW

**Comment by dpshelio** [comment]
Ping @jni

File: doc/examples/plot_marked_watershed.py [hunk 1]
```diff
@@ -28,11 +32,12 @@
 # denoise image
 denoised = rank.median(image, disk(2))
 
-# find continuous region (low gradient) --> markers
+# find continuous region (low gradient - where less than 10 for this image) --> markers
```
Comment (jni) [inline doc/examples/plot_marked_watershed.py]: Sorry about this one — lines should not be longer than 79 characters!

File: doc/examples/plot_marked_watershed.py [hunk 2]
```diff
@@ -28,11 +32,12 @@
...
+# disk(5) is used here to get a more smooth image
```
Comment (jni) [inline doc/examples/plot_marked_watershed.py]: more smooth -> smoother

**Comment by jni** [comment]
@oew1v07 Great stuff! Titles on the images are particularly valuable. Just a couple of more final fixes and it'll be ready to merge!

File: doc/examples/plot_marked_watershed.py (hunk 1 above)
Comment (dpshelio) [inline doc/examples/plot_marked_watershed.py]: This was left on purpose for clarity

Comment (emmanuelle) [inline doc/examples/plot_marked_watershed.py]: sure, but can you break the line?

**Comment by oew1v07** [comment]
@jni @emmanuelle Split the line into two so that it made sense

**Pull request merged**